skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md

```diff
@@ -39,12 +37,20 @@ This cmdlet returns a list of all blocked number patterns added to the tenant li
 PS C:> Get-CsInboundBlockedNumberPattern

```

Comment:
The prompt should be ```PS>``` per https://docs.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.2#avoid-using-powershell-prompts-in-examples.